core/src/main/java/hudson/PluginManager.java

```diff
@@ -446,6 +478,40 @@ public void run(Reactor reactor) throws Exception {
         }});
     }
 
+    @Extension
+    public final static PluginManagerAdministrativeMonitor NOTICE = new PluginManagerAdministrativeMonitor();
```

Comment:
>  I guess that because they are not loaded at this point 

It's not clear why this needs to run so early during initialization. This is only for UI use, right? So why not wait until plugins and extensions are done loading?